public/docs/ts/latest/glossary.jade

```diff
@@ -95,13 +93,13 @@ block includes
       import { Hero }          from '../heroes/hero.model.ts';
       import { HeroService }   from '../heroes/hero.service.ts';
     :marked
-      We can add a barrel to the `heroes` folder (called `index` by convention) that exports all of these items:
+      You can add a barrel to the `heroes` folder (called `index`, by convention) that exports all of these items:
     code-example.
       export * from './hero.model.ts';   // re-export all of its exports
       export * from './hero.service.ts'; // re-export all of its exports
       export { HeroComponent } from './hero.component.ts'; // re-export the named thing
     :marked
-      Now a consumer can import what it needs from the barrel.
+      Now a consumer can import what they need from the barrel.
```

Comment:
Change back to "it needs". We're talking about a piece of software here, not a person.